zfs/replicate/compress/__init__.py

```diff
@@ -1,4 +1,4 @@
 """ZFS Replication Stream Compression."""
 
-from .command import command
-from .type import Compression
+from .command import command as command  # pylint: disable=useless-import-alias
```

Comment:
Maybe disable useless-import-alias on the `setup.cfg` instead?

Reply:
I usually like leaving things explicit but I'll consider that.  What's the state of the art on linting Python code?  Should I just drop pylint in favour of something else instead?

Reply:
I've been mostly relying on flake8 only, configured with [pre-commit](https://pre-commit.com) together with isort, black and reuse.

Reply:
I'll setup a new pull request to change to flake8 with pre-commit.  Thanks!